### PR TITLE
Adding functionality to look up who owns a player

### PR DESCRIPTION
**Pull request by DMcP89**

### CODE REVIEW

File: yahoo_fantasy_api/league.py [hunk 1]
```diff
@@ -552,6 +552,30 @@ def percent_owned(self, player_ids):
...
+    def ownership(self, player_ids):
+        """Retrieve the owner of a player
+
+        :param player_ids: Yahoo! Player IDs to retrieve owned for
+        :type player_ids: list(int)
+        :return: Ownership status of player
+        :rtype: dict
+
+        >>> lg.player_owner(1, [3737])
+        {"3737" : {"ownership_tpye" : "team", "owner_team_name": "team name"}}
```
Comment (spilchen) [inline yahoo_fantasy_api/league.py:565]: I like that you have included sample code, but it seems like it is out of date.  Can you update it to reflect the current API name?

**Review by spilchen — COMMENTED** [review]
I like the new API.  Just had a minor comment about the sample.

**Comment by DMcP89** [comment]
@spilchen updated the sample to be up to date with the API

File: yahoo_fantasy_api/league.py [hunk 2]
```diff
@@ -552,6 +552,30 @@ def percent_owned(self, player_ids):
...
+
+    def ownership(self, player_ids):
+        """Retrieve the owner of a player
+
+        :param player_ids: Yahoo! Player IDs to retrieve owned for
+        :type player_ids: list(int)
+        :return: Ownership status of player
+        :rtype: dict
+
+        >>> lg.ownership(1, [3737])
```
Comment (spilchen) [inline yahoo_fantasy_api/league.py:564]: The first parm in the sample here (`1`) doesn't exist.  Can you remove it?

**Comment by DMcP89** [comment]
@spilchen fixed that typo

**Review by spilchen — APPROVED** [review]

**Pull request merged**